corehq/apps/export/export.py

```diff
@@ -285,8 +286,8 @@ def get_export_download(export_instances, filters, filename=None):
 
     download = DownloadBase()
     download.set_task(populate_export_download_task.delay(
-        export_instances,
-        filters,
+        pickle.dumps(export_instances),
```

Comment:
I think this may not be necessary if you set `populate_download_task` to have serializer=pickle

Reply:
> I think this may not be necessary if you set populate_download_task to have serializer=pickle

This is true, but in the interests of moving away from pickle, I'd prefer explicitly pickling individual arguments and using the JSON serializer for the arguments as a whole.  Since this work is already done, feels worth merging as-is.  It means one less task will need ```serializer='pickle'```.